package.json

```diff
@@ -44,9 +44,9 @@
 	"dependencies": {
 		"chalk": "^1.1.3",
 		"ecmarkdown": "^6.0.0",
-		"eslint": "^7.25.0",
+		"eslint": "^7.28.0",
```

Comment:
I had to update the linter and fix a bunch of linting issues because it was incompatible with the new TypeScript version.